.travis.yml

```diff
@@ -36,16 +36,15 @@ jobs:
         - name: "Run end-to-end tests"
           dist: trusty
           addons:
-              chrome: stable
+              firefox: latest
           services:
               - xvfb
           before_install:
-              - export CHROME_BIN=chromium-browser
               - export DISPLAY=:99.0
               - sh -e /etc/init.d/xvfb start
               - sleep 3
           script:
-              - travis_wait 30 npm run test:e2e -- $SINCE_ARG
+              - npm run test:e2e -- $SINCE_ARG
```

Comment:
We don't need this line anymore because we have a script to heat up the sandbox. This was used before to wait for the sandbox indefinitely without response.